### PR TITLE
feat: enable rss feeds

### LINKED ISSUE #62: Generate RSS XML data for RSS Reader
# Description

* 增加订阅 feed 信息

# Solution
使用插件

https://github.com/hexojs/hexo-generator-feed

自动构建后，从 https://flossway.github.io/flossway/rss2 获得 Feeds

Former Issue is for sitemap. 

https://github.com/flossway/flossway/issues/58

### PR DESCRIPTION
**Pull request by hailiang-wang**
* Add rss generator plugin for rss feeds

This closes #62.

### CODE REVIEW

**Comment by hailiang-wang** [comment]
Now, it works.

The preview url https://flossway-cuz6bhyfe-flossway.vercel.app/atom.xml

Subscribe in the reader - 

image

File: _config.yml [hunk 1]
```diff
@@ -139,3 +139,19 @@ sitemap:
...
+  path: atom.xml
+  limit: 0
+  hub:
+  content: true
+  content_limit: 140
+  content_limit_delim: ' '
+  order_by: -date
+  icon: /flossway/images/favicon-32x32-next.png
+  autodiscovery: true
+  template:
```
Comment (tisonkun) [inline _config.yml:157]:
```suggestion

```

File: _config.yml [hunk 2]
```diff
@@ -139,3 +139,18 @@ sitemap:
...
+  type: atom
+  path: atom.xml
+  limit: 0
+  hub:
+  content: true
+  content_limit: 140
+  content_limit_delim: ' '
+  order_by: -date
+  icon: /flossway/images/favicon-32x32-next.png
+  autodiscovery: true
```
Comment (tisonkun) [inline _config.yml:156]:
```suggestion
  autodiscovery: true

```

Comment (tisonkun) [reply _config.yml:156]:
```suggestion
  autodiscovery: true

```

**Review by tisonkun — APPROVED** [review]

**Pull request merged**